modules/tensor_mechanics/src/kernels/WeakPlaneStress.C

```diff
@@ -144,9 +166,13 @@ WeakPlaneStress::computeQpOffDiagJacobian(unsigned int jvar)
   // off-diagonal Jacobian with respect to a coupled temperature variable
   if (_temp_coupled && jvar == _temp_var)
```

Comment:
if temperature is not coupled then `jvar` will never be `_temp_var`

Reply:
So you would just change this to read:
```
if (jvar == _temp_var)
```
correct?  No objections to that, but kind of outside the scope of this PR.